hostsfile/init.sls

```diff
@@ -10,13 +10,15 @@
 #  mine_interval: 2
 
 {%- set minealias = salt['pillar.get']('hostsfile:alias', 'network.ip_addrs') %}
-{%- set minion_filter = salt['pillar.get']('hostsfile:filter', '*') %}
-
-{%- set addrs = salt['mine.get'](minion_filter, minealias) %}
+{%- set minions = salt['pillar.get']('hostsfile:minions', '*') %}
+{%- set hosts = salt['pillar.get']('hostsfile:hosts', {}) %}
 
+{%- set addrs = salt['mine.get'](minions, minealias) %}
 {%- if addrs is defined %}
+{%- do hosts.update(addrs) %}
```

Comment:
Pretty sure it overrides. The entries in mine would take priority.

Perhaps it should be the other way around. I hadn't thought about that.